docs/networking.md

```diff
@@ -459,6 +457,14 @@ $ kops create cluster \
 
 You can adjust Cilium agent configuration with most options that are available in [cilium-agent command reference](http://cilium.readthedocs.io/en/stable/cmdref/cilium-agent/).
 
+The following command will create your cluster with desired Cilium configuration
+
+```console
+$ kops update cluster myclustername.mydns.io --yes
```

Comment:
That updates a cluster, not creates.

Reply:
This is from the original documentation. I just moved this a bit up. 
And well–on first run this does creates all the AWS resources and such.